templates/sheets/actor/monster-sheet.hbs

```diff
@@ -313,6 +313,9 @@
                 <div class="tab" data-group="primary" data-tab="details">
                     {{>"systems/TheWitcherTRPG/templates/partials/monster/monster-details-tab.hbs"}}
                 </div>
+                <div class="tab" data-group="primary" data-tab="details">
+                    {{> "systems/TheWitcherTRPG/templates/partials/monster/monster-profession-skill-display.hbs"}}
+                </div>
```

Comment:
This should not work. If it should be on the same tab as monster-details you can just append the partial below the details partial